infra/integration-image/Dockerfile.test

```diff
@@ -57,3 +57,31 @@ RUN curl -fsSLO https://releases.hashicorp.com/packer/0.12.2/packer_0.12.2_linux
 
 RUN curl -sL https://deb.nodesource.com/setup_7.x | bash -
 RUN apt-get install -y nodejs
+
+# Add docker in docker support
```

Comment:
You need to push this container to the repo `gcr.io/eminent-nation-87317/vic-integration-test:xx` and bump up the URL in all locations, including secrets for this to be picked up